apps/rebar/src/rebar_utils.erl

```diff
@@ -69,7 +69,7 @@
          escape_chars/1,
          escape_double_quotes/1,
          escape_double_quotes_weak/1,
-         check_min_otp_version/1,
+         check_min_otp_version/2,
```

Comment:
we likely want to keep a copy of `check_min_otp_version/1` because it's possible plugins started relying on it over the years, and removing the function may break builds in the ecosystem.

Reply:
You are right 😄 I'll add it back.

Reply:
0a9a43b6